arm-recoveryservicesbackup/2016-12-01/swagger/backupManagement.json

```diff
@@ -18,35 +18,6 @@
     "application/json"
   ],
   "paths": {
-    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/operations": {
```

Comment:
seeing that you also fixed the `x-ms-pageable` thing (`itemName`), I assume this wasn't released yet?